dev-libs/userspace-rcu/files/userspace-rcu-0.13.1-loong.patch

```diff
@@ -0,0 +1,203 @@
+From 9527dea023eb06e49569da12fd286e7e7f568cbb Mon Sep 17 00:00:00 2001
```

Comment:
Please always include links to where patches come from, upstream submissions, etc.

Also, run scrub-patch.

Reply:
Fixed.